worker_tasks/scraper.py

```diff
@@ -11,17 +13,21 @@
 
 import logging
 
-def scrape_articles(categorizer):
+logger = logging.getLogger('Sourcemash')
+
+def scrape_and_categorize_articles():
+    categorizer = Categorizer()
```

Comment:
To confirm, we can/are doing this because we're now using Wikipedia as the universe of possible categories and thus not maintaining an application-level notion of categories like we were before these changes?